lms/static/scripts/frontend_apps/config.ts

```diff
@@ -279,6 +279,15 @@ export type DashboardUser = {
 export type DashboardConfig = {
   routes: DashboardRoutes;
   user: DashboardUser;
+
+  /** Whether syncing grades is enabled for auto-grading assignments or not */
+  auto_grading_sync_enabled: boolean;
```

Comment:
This is not part of this PR specifically, but I'm adding it to the type definition to reduce conflicts with other branches later.